services/brig/src/Brig/Provider/API.hs

```diff
@@ -816,7 +816,7 @@ addBot zuid zcon cid add = do
     throwStd tooManyMembers
   -- For team conversations: bots are not allowed in managed and in
   -- team-only conversations
-  when (cnvAccessRole cnv == TeamAccessRole) $
+  unless (Set.member ServiceAccessRole (cnvAccessRoles cnv)) $
```

Comment:
This is the reason for the failing tests in brig.

Reply:
Yes, this is fixed now after adding `ServiceAccessRole` to the mapping of `ActivedAccessRole`.